doc/usage/unitpackage_usage.md

```diff
@@ -61,6 +61,21 @@ filtered_db = db.filter(custom_filter)
 len(filtered_db)
 ```
 
+A new collection from an existing collection can be created from a lists of selected identifiers
+
+```{code-cell} ipython3
+ids_db = db['engstfeld_2018_polycrystalline_17743_f4b_1','alves_2011_electrochemistry_6010_f1a_solid']
```

Comment:
Is a list of indices also possible?

Reply:
Yes. I added an example. Various possibilities are included in the documentation of the API